benchmarks/2019.json

```diff
@@ -5455,7 +5436,7 @@
     "deciles": [
       0,
       56.93,
-      58.60,
```

Comment:
The deciles here got changed by my IDE possibly. Will this negatively effect us?

Reply:
To be safe please restore these to the original values

Reply:
This has been restored.